src/main/java/com/google/devtools/build/lib/bazel/bzlmod/IndexRegistry.java

```diff
@@ -117,18 +118,51 @@ private static class BazelRegistryJson {
     String moduleBasePath;
   }
 
-  /** Represents fields available in {@code source.json} for each version of a module. */
+  /** Represents the type field in {@code source.json} for each version of a module. */
   private static class SourceJson {
     String type = "archive";
```

Comment:
Should we remove the default value here?

Reply:
I do not think so. The documentation states that the default type is "archive".
https://github.com/bazelbuild/bazel/blob/master/site/en/external/registry.md?plain=1#L57

Reply:
SG